ui/lib/kmip/addon/templates/credentials/show.hbs

```diff
@@ -31,30 +31,29 @@
     <ToolbarLink @route="credentials.index" @models={{array this.scope this.role}} data-test-kmip-link-back-to-role>
       Back to role
     </ToolbarLink>
-    <CopyButton
-      class="toolbar-link"
-      @clipboardText={{this.model.certificate}}
-      @success={{action (set-flash-message "Certificate Copied!")}}
+    <Hds::Copy::Button
+      @text="Copy certificate"
+      @textToCopy={{this.model.certificate}}
+      class="toolbar-link is-flex-center"
       data-test-copy-button
-    >
-      Copy certificate
-      <Chevron />
-    </CopyButton>
+    />
   </ToolbarActions>
 </Toolbar>
 <div class="box is-shadowless is-fullwidth is-sideless">
   <InfoTableRow @label="Serial number" @value={{this.model.id}}>
     <MaskedInput @value={{this.model.id}} @displayOnly={{true}} @allowCopy={{true}} />
   </InfoTableRow>
-  <InfoTableRow @label="Private key" @value={{this.model.privateKey}}>
-    <div class="is-block">
-      <Hds::Alert data-test-warning @type="inline" @color="warning" @class="has-bottom-margin-s" as |A|>
-        <A.Title>Warning</A.Title>
-        <A.Description>You will not be able to access the private key later, so please copy the information below.</A.Description>
-      </Hds::Alert>
-      <MaskedInput @value={{this.model.privateKey}} @name="Private key" @allowCopy={{true}} @displayOnly={{true}} />
-    </div>
-  </InfoTableRow>
+  {{#if this.model.privateKey}}
```

Comment:
added this conditional because this row was always rendering, even when there was no private key